### PR TITLE
link the runtime graph in the lock file

### PR DESCRIPTION
**Pull request by davidfowl**
I need to add a test for this

https://github.com/NuGet/Home/issues/1890

/cc @ericstj @emgarten 

e.g.:

``` JSON
"System.Runtime.Extensions/4.0.11-beta-23516": {
    "type": "package",
    "dependencies": {
      "System.Runtime": "4.0.20",
      "runtime.win7.System.Runtime.Extensions": "4.0.11-beta-23516"
    },
    "compile": {
      "ref/dotnet5.4/System.Runtime.Extensions.dll": {}
    }
},
```

### CODE REVIEW

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs [hunk 1]
```diff
@@ -30,7 +30,11 @@ public static int Main(string[] args)
             if (args.Contains("--debug"))
             {
                 args = args.Skip(1).ToArray();
-                Debugger.Launch();
```
Comment (davidfowl) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs:33]: This breaks core clr

Comment (ericstj) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs:33]: Debugger.Launch not working looks like a separate bug.  Do runtime folks know about this?

**Comment by ericstj** [comment]
Will this cause any new graph traversals during cycle detection?  Trying to gauge if we might expect expect new circular dependencies out of this support.

Otherwise LGTM.

**Comment by emgarten** [comment]
:shipit:

**Comment by emgarten** [comment]
Merged with NuGet/NuGet.Client@6f8f8e783fc2f314c5b27498225d9e4db006c0da

**Pull request closed without merging**